src/librustdoc/passes/strip_private.rs

```diff
@@ -17,13 +17,12 @@ crate const STRIP_PRIVATE: Pass = Pass {
 crate fn strip_private(mut krate: clean::Crate, cx: &mut DocContext<'_>) -> clean::Crate {
     // This stripper collects all *retained* nodes.
     let mut retained = DefIdSet::default();
-    let access_levels = cx.renderinfo.borrow().access_levels.clone();
 
     // strip all private items
     {
         let mut stripper = Stripper {
             retained: &mut retained,
-            access_levels: &access_levels,
+            access_levels: &cx.cache.access_levels,
```

Comment:
Note this removes the `clone()` since it wasn't necessary; since modifying `access_levels` requires `&mut Cache`, we know now that the interior mutability is gone, it can't be modified in `ImportStripper`.